### ISSUE TITLE
iconSetsUrl is treated as a directory path, not a URL

### ISSUE THREAD

**Issue by michaelroper** [verbb/icon-picker#9]
Hey Josh! just a heads up because I just got bit by something similar to #8 ... 

Because you are passing the `iconSetsUrl` variable through the `FileHelper::normalisePath` function ([like here](https://github.com/verbb/icon-picker/blob/44d1c310af76cbcc9e1934d778aaefa3128fcdeb/src/variables/IconPickerVariable.php#L36)), it actually break the image URL if you set `iconSetsUrl` to an actual fully qualified URL, because it converts double slashes to a single slash, breaking any `https://...` in the URL.

tl;dr:

✅ `'iconSetsUrl' => '/icons/'`

📛 `'iconSetsUrl' => getenv('SITE_URL') . 'icons/'`

📛 `'iconSetsUrl' => 'https://my.file.cdn.com/icons/'` 
(URL becomes `https:/my.file.cdn.com/icons/`)

**Comment by engram-design** [comment]
Yeah, I knew `normalisePath` might not be the proper function for this case. All I wanted was to ensure the path had the proper slashes. As in, these should all be treated the same:

```
'iconSetsUrl' => 'icons/'
'iconSetsUrl' => 'icons'
'iconSetsUrl' => '/icons'
'iconSetsUrl' => '/icons/'
```
I'll see if I can throw together a better function for this.

**Comment by michaelroper** [comment]
hah, yeah.. this is surely a problem that has been solved before? 😛

**Comment by engram-design** [comment]
Fixed in [1.0.5](https://github.com/verbb/icon-picker/releases/tag/1.0.5)
